python_modules/libraries/dagstermill/dagstermill/examples/repository.py

```diff
@@ -72,7 +72,14 @@ def test_nb_solid(name, **kwargs):
     )
 
 
-default_mode_defs = [ModeDefinition(resource_defs={"file_manager": local_file_manager})]
+default_mode_defs = [
+    ModeDefinition(
+        resource_defs={
+            "file_manager": local_file_manager,
```

Comment:
After you make the changes in your other PR, it would be nice to get rid of file manager in this example.  Having both is confusing.

Reply:
working on it in https://github.com/dagster-io/dagster/pull/4490